src/Components/Admin/index.js

```diff
@@ -0,0 +1,138 @@
+import React, {Component} from 'react';
+import { thisExpression } from '@babel/types';
+
+class Admin extends Component {
+    check(e){
+        console.log(e.target);
+    }
+    state = {
+        events: [
```

Comment:
for now I predefined events array, Ill take it from local storage if rest of the code is fine.